CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 2.8)
-project(AimTux)
+project(Fuzion) # Why was this "AimTux" when you add libraries to "Fuzion"?
```

Comment:
Wasn't really concerned with changing every instance of AimTux to Fuzion.

I used to actually sort of pay homage to the original AimTux ( hence the name AimTux-Fuzion ) but mcswaggens is kind of a dick